jwst/datamodels/schemas/container.schema.yaml

```diff
@@ -16,32 +11,14 @@ properties:
         title: Name of ASN pool used to generate this table
         type: string
         fits_keyword: ASNPOOL
```

Comment:
For now let's just remove the ``fits_keyword`` definitions for ASNTAB and ASNPOOL, so that they're only defined in core.schema.yaml. You can keep the meta data objects for use within ``ModelContainer``'s.